docs/markdown/node/BaseClient/classes/BaseClient.md

```diff
@@ -0,0 +1,8382 @@
+[**@valkey/valkey-glide**](../../README.md)
+
+***
+
+[@valkey/valkey-glide](../../modules.md) / [BaseClient](../README.md) / BaseClient
+
+# Class: BaseClient
+
+## Extended by
+
+- [`GlideClient`](../../GlideClient/classes/GlideClient.md)
+- [`GlideClusterClient`](../../GlideClusterClient/classes/GlideClusterClient.md)
+
+## Constructors
+
+### new BaseClient()
+
+> `protected` **new BaseClient**(`socket`, `options`?): [`BaseClient`](BaseClient.md)
+
+**`Internal`**
+
+#### Parameters
+
+| Parameter | Type |
+| ------ | ------ |
+| `socket` | `Socket` |
+| `options`? | [`BaseClientConfiguration`](../interfaces/BaseClientConfiguration.md) |
+
+#### Returns
+
+[`BaseClient`](BaseClient.md)
+
+## Properties
+
+| Property | Modifier | Type | Default value |
+| ------ | ------ | ------ | ------ |
+| <a id="defaultdecoder"></a> `defaultDecoder` | `protected` | [`Decoder`](../enumerations/Decoder.md) | `Decoder.String` |
+| <a id="isclosed"></a> `isClosed` | `protected` | `boolean` | `false` |
+| <a id="promisecallbackfunctions"></a> `promiseCallbackFunctions` | `readonly` | \[`PromiseFunction`, `ErrorFunction`, `undefined` \| [`Decoder`](../enumerations/Decoder.md)\][] \| \[`PromiseFunction`, `ErrorFunction`\][] | `[]` |
+
+## Methods
+
+### append()
+
+> **append**(`key`, `value`): `Promise`\<`number`\>
```

Comment:
All that line (and all similar ones) should be in a code block for readability.
Something like
```suggestion
> `append(key, value): Promise<number>`
```